src/pages/[platform]/build-a-backend/storage/download-files/index.mdx

```diff
@@ -40,19 +42,187 @@ export const DefaultStorageImageExample = () => {
 
 Learn more about how you can further customize the UI component by referring to the [Storage Image documentation](https://ui.docs.amplify.aws/react/connected-components/storage/storagemanager).
 
-#### API Usage
 To further customize your in-app experience, you can use the `getUrl` or `downloadData` API from the Amplify Library for Storage.
 
-<Columns columns={2}>
-<Card variation="outlined">
-[Get URL API usage](/[platform]/build-a-backend/storage/download-files/get-url)
-</Card>
-<Card variation="outlined">
-[Download data API usage](/[platform]/build-a-backend/storage/download-files/download-data)
-</Card>
-</Columns>
+## API to get or download file from URL
+
+Get a presigned URL of a stored file and expiration of URL
+
+```typescript
+import { getUrl } from 'aws-amplify/storage';
+
+const getUrlResult = await getUrl({
+  path: "album/2024/1.jpg",
+  // Alternatively, path: ({identityId}) => `album/{identityId}/1.jpg`
+  options: {
+    validateObjectExistence?: false,  // defaults to false
+    expiresIn?: 20 // validity of the URL, in seconds. defaults to 900 (15 minutes) and maxes at 3600 (1 hour)
+    useAccelerateEndpoint: true; // Whether to use accelerate endpoint.
+  },
+});
+console.log('signed URL: ', getUrlResult.url);
+console.log('URL expires at: ', getUrlResult.expiresAt);
+```
+
+`getUrl` returns a signed URL in the `url` property of the result. You can use this to create a download link for users to click on. The `expiresAt` property is a `Date` object that represents the time at which the URL will expire.
+
+Inside your template or JSX code, you can use the `url` property to create a link to the file:
+
+```tsx
+<a href={signedURL.url.toString()} target="_blank" rel="noreferrer">
+  {fileName} 
+</a>
```

Comment:
Is this missing the closing backticks to close out the codeblock?

Reply:
Yes, thanks for pointing it out. Updating it.